### PR TITLE
Fix getting started app

### LINKED ISSUE #3078: Getting started example does not work
I get a 'table blog does not exist' error and I verified the table was created correctly in the blogging.db.  I've been trying to use sqlite with entity framework and I simply cannot get it to work.  Not a fan of EF.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 488629d4-9bd2-42b5-4644-34e3f2baee59
* Version Independent ID: b38b9d14-6bad-12e0-8d2d-24b50bc4977e
* Content: [Getting Started - EF Core](https://docs.microsoft.com/en-us/ef/core/get-started/overview/first-app?tabs=netcore-cli)
* Content Source: [entity-framework/core/get-started/overview/first-app.md](https://github.com/dotnet/EntityFramework.Docs/blob/master/entity-framework/core/get-started/overview/first-app.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **avickers**

### PR DESCRIPTION
**Pull request by roji**
Fixes #3078

### CODE REVIEW

**Review by ajcvickers — CHANGES_REQUESTED** [review]
See comment on issue.

File: samples/core/GetStarted/Model.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class BloggingContext : DbContext
         public DbSet<Post> Posts { get; set; }
 
         protected override void OnConfiguring(DbContextOptionsBuilder options)
-            => options.UseSqlite("Data Source=blogging.db");
+            => options.UseSqlite(@"Data Source=C:\blogging.db");
```
Comment (Rick-Anderson) [inline samples/core/GetStarted/Model.cs:12]: The VS Mac tab is available but probably not what you want. Need either code comments or note before code for mac/linix

Comment (roji) [reply samples/core/GetStarted/Model.cs:12]: How does this note look?

**Review by Rick-Anderson — APPROVED** [review]

File: samples/core/GetStarted/Model.cs (hunk 1 above)
Comment (smitpatel) [reply samples/core/GetStarted/Model.cs:12]: Can also add a comment here in code what to use if your are using mac/linux.

Comment (Rick-Anderson) [reply samples/core/GetStarted/Model.cs:12]: I agree, it turns out even folks reading localized content prefer helpful english code comments. I hate the chrome of a note. I'd move this to the code comments and remove the note. Note chrome detracts from the flow of the article.

Comment (roji) [reply samples/core/GetStarted/Model.cs:14]: Changed to note into a code comment

**Review by ajcvickers — APPROVED** [review]

**Pull request merged**